src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -53,6 +53,8 @@ public final class GATKVCFConstants {
     public static final String GQ_STDEV_KEY =                       "GQ_STDDEV";
     public static final String HAPLOTYPE_SCORE_KEY =                "HaplotypeScore";
     public static final String HI_CONF_DENOVO_KEY =                 "hiConfDeNovo";
+    public static final String TRANSMITTED_SINGLETON =              "transmittedSingleton";
```

Comment:
Generally these are captialized... i don't know if this breaks already run samples however...

Reply:
This is low priority and you know better than i do. Feel free to ignore this comment I you like.